### PR TITLE
congestion: script to reproduce fine-tuning data

### PR DESCRIPTION
**Pull request by jakmeier**
This script was used to produce the data in https://docs.google.com/spreadsheets/d/1Vt_-sgMdX1ncYleikYY8uFID_aG9RaqJOqaVMLQ37tQ/edit#gid=0.

It contains the set of model runs that I found informative to come up with a proposal for the fine-tuned parameters.
Maybe other people can use it to quickly compare their improvement ideas, I'm sure we haven't found the "best" strategy, yet, if such a thing exists.

I think it makes sense to store it along the source. But I am also open to just storing on the drive and not adding it to git repo.

### CODE REVIEW

File: tools/congestion-model/run_eval.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+NUM_SHARDS=6
+WARMUP=500
+ROUNDS=1500
+TX_POOL_SIZE=200
+
+OPTIONS_STRING="--rounds ${ROUNDS} --warmup ${WARMUP} --shards ${NUM_SHARDS} --tx-pool-size ${TX_POOL_SIZE}"
+
+echo "# Delay VS Utilization"
+cargo run -q --release -- --workload RelayedHot --strategy SmoothTrafficLight ${OPTIONS_STRING}
```
Comment (wacban) [inline tools/congestion-model/run_eval.sh:17]: Why --release?

File: tools/congestion-model/run_eval.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# exit on error
+set -euo pipefail
```
Comment (wacban) [inline tools/congestion-model/run_eval.sh:4]: TIL, nice!
I typically only use -e, this is better!

**Review by wacban — APPROVED** [review]
LGTM

File: tools/congestion-model/run_eval.sh (hunk 1 above)
Comment (jakmeier) [reply tools/congestion-model/run_eval.sh:17]: Because debug builds are a lot slower in execution.

**Pull request merged**